src/components/timeline/form/TimelineEditorLayerFilter.svelte

```diff
@@ -39,6 +41,19 @@
     filteredValues = options.slice();
   }
 
+  $: if ($activityTypes || $externalResourceNames) {
```

Comment:
Do you mind moving this filtering logic back up to the parent component `TimelineEditorPanel`? `TimelineEditorPanel` is already doing logic to derive values to pass into `TimelineEditorLayerFilter` so I think it'd be cleaner to not split that logic into both the parent and the child.

Reply:
Done, thanks for the tip for reactivity!